apps/server/src/modules/learnroom/repo/mikro-orm/dashboard.spec.ts

```diff
@@ -1,88 +1,82 @@
 import { BadRequestException, NotFoundException } from '@nestjs/common';
-import { LearnroomTypes } from '@shared/domain/types';
-import { Learnroom } from '../interface';
-import { DashboardEntity, GridElement } from './dashboard';
-
-const getLearnroomMock = (id: string): Learnroom => {
-	return {
-		getMetadata: () => {
-			return {
-				id,
-				type: LearnroomTypes.Course,
-				title: 'Reference',
-				shortTitle: 'Re',
-				displayColor: '#FFFFFF',
-				isSynchronized: false,
-			};
-		},
-	};
-};
+import { Course, CourseGroup } from '@shared/domain/entity';
+import { setupEntities } from '@testing/database';
+import { courseFactory } from '@testing/factory/course.factory';
```

Comment:
Wenn jetzt die api vom course hier liegt. Wäre dann ein mögliches weiteres Ticket das CourseRepo, CourseEntity auch hierhin zu verschieben? Ein Vorschlag von @Metauriel learnroom umzubenennen, da es eigentlich nur Course Infos darstellt. Evtl. mit anderen Sichten drauf. 
Oder es sind doch 2 Module die aber stark verzahnt sind, aber auseinander müssen. 
Der Course als Fachlichkeit ist ja auch in anderen Kontexten Notwendig ohne das ganze Dashboard zeug.